src/__tmpdata/services.json

```diff
@@ -8,27 +8,107 @@
     "title": "Use City of Austin Services",
     "body": "<p>The City of Austin provides hundreds of services to people. This is a short list of services that will grow over time.</p>"
   },
-  "contact":{
-    "phone":"512-367-8695",
-    "email":"resourcerecovery@austintexas.gov",
-    "address": {
+  "contacts":[
+    {
       "name": "Austin Recycle and Reuse Drop-off Center",
-      "street": "2514 Business Center Drive ",
-      "city": "Austin",
-      "state": "Texas",
-      "zip": "78744 ",
-      "country": "United States"
+      "phone":"512-367-8695",
+      "email":"resourcerecovery@austintexas.gov",
+      "address": {
+        "street": "2514 Business Center Drive",
+        "city": "Austin",
+        "state": "Texas",
+        "zip": "78744",
+        "country": "United States"
+      },
+      "hours": [
```

Comment:
There's a slightly different structure for contacts & locations currently https://github.com/cityofaustin/joplin/issues/39#issuecomment-354149336. Some goofiness is there because of the default Django rest framework structure, but I separated contact & location. Not sure if that's the right way to model it or not.

Reply:
made some comments here -- https://github.com/cityofaustin/joplin/issues/39
but basically, since there will be multiple locations and points of contact, I think there will be cases where these two should be associated when shown to residents. For ex, contact info for Austin Resource Recovery main drop off location vs Treehouse where classes are hosted.